pom.xml

```diff
@@ -50,10 +49,7 @@
         <maven-source-plugin.version>3.3.0</maven-source-plugin.version>
         <maven-surefire-plugin.version>3.2.3</maven-surefire-plugin.version>
         <maven.compiler.release>11</maven.compiler.release>
```

Comment:
If we're going to build for SB 3, then 11 makes no more sense really. The baseline is 17.